code/web/src/modules/user/test/profile.test.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import '@testing-library/jest-dom'
+import userEvent from '@testing-library/user-event'
+import Profile from '../profile.js'
+import { profileState } from '../profile.js'
+import { BrowserRouter } from 'react-router-dom'
+import { Router } from 'react-router-dom'
+import { createStore } from 'redux'
+import rootReducer from '../../../setup/store.js'
+import { Provider } from 'react-redux'
+import configureStore from 'redux-mock-store'
+import renderer from 'react-test-renderer'
+
+const mockStore = configureStore([])
+
+describe("Profile", () => {
+  let store;
+  let component;
```

Comment:
Are these 2 lines not "equal" to anything because you begin by defining a variable?